src/99_appendices/03_hed.md

```diff
@@ -1,36 +1,46 @@
 # Appendix III: Hierarchical Event Descriptor (HED) Tags
 
-Each event can be assigned one or more Hierarchical Event Descriptor (HED) Tag (see [https://github.com/BigEEGConsortium/HED/wiki/HED-Schema](https://github.com/BigEEGConsortium/HED-schema/wiki/HED-Schema) for more details) under the optional `HED` column.
-HED is a controlled vocabulary of terms describing events in a behavioural paradigm. It was originally developed with EEG in mind, but is applicable to all behavioural experiments. Each level of the hierarchical tags are delimited with a forward slash ("/"). Multiple tags are delimited with a comma. Parentheses (brackets) group tags and enable specification of multiple items and their attributes in a single HED string (see section 2.4 in [HED Tagging Strategy Guide](http://www.hedtags.org/downloads/HED%20Tagging%20Strategy%20Guide.pdf)). For more information about HED and tools available to validate and match HED strings, please visit [www.hedtags.org](http://www.hedtags.org).
+Events in a given study are often annotated using study-specific codes, making it difficult to compare events across studies. Hierarchical Event Descriptor (HED) Tags (see [https://github.com/BigEEGConsortium/HED/wiki/HED-Schema](https://github.com/BigEEGConsortium/HED-schema/wiki/HED-Schema) for more details) provide a mechanism for annotating events in a study-independent way. 
 
-Since dedicated fields already exist for the overall task classification (`CogAtlasID` and `CogPOID`) in the sidecar JSON files HED
-tags from the Paradigm subcategory should not be used to annotate
-events.
+HED is a controlled vocabulary of terms describing events in a behavioural paradigm. HED was originally developed with EEG in mind, but is applicable to all behavioural experiments. Each level of a hierarchical tag is delimited with a forward slash ("/"). A HED string contains one or more HED tags separated by commas. Parentheses (brackets) group tags and enable specification of multiple items and their attributes in a single HED string (see section 2.4 in [HED Tagging Strategy Guide](http://www.hedtags.org/downloads/HED%20Tagging%20Strategy%20Guide.pdf)). For more information about HED and tools available to validate and match HED strings, please visit [www.hedtags.org](http://www.hedtags.org). Since dedicated fields already exist for the overall task classification (`CogAtlasID` and `CogPOID`) in the sidecar JSON files HED tags from the Paradigm subcategory should not be used to annotate events.  
```

Comment:
```suggestion
HED is a controlled vocabulary of terms describing events in a behavioural paradigm. HED was originally developed with EEG in mind, but is applicable to all behavioural experiments. Each level of a hierarchical tag is delimited with a forward slash ("/"). A HED string contains one or more HED tags separated by commas. Parentheses (brackets) group tags and enable specification of multiple items and their attributes in a single HED string (see section 2.4 in [HED Tagging Strategy Guide](http://www.hedtags.org/downloads/HED%20Tagging%20Strategy%20Guide.pdf)). For more information about HED and tools available to validate and match HED strings, please visit [www.hedtags.org](http://www.hedtags.org). Since dedicated fields already exist for the overall task classification (`CogAtlasID` and `CogPOID`) in the sidecar JSON files, HED tags from the Paradigm subcategory should not be used to annotate events.  
```

Reply:
note, I just added a comma between 

> ... in the sidecar JSON files

and

>  ... HED tags from the